HeadlessUI/HeadlessUI.csproj

```diff
@@ -4,9 +4,12 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Adding this because of:
https://docs.microsoft.com/en-us/aspnet/core/blazor/components/class-libraries?view=aspnetcore-5.0&tabs=visual-studio#browser-compatibility-analyzer-for-blazor-webassembly